src/crc32c/__init__.py

```diff
@@ -42,3 +44,57 @@ def value(chunk):
         int: New CRC checksum computed for ``chunk``.
     """
     return crc32c._crc32c_cffi.lib.crc32c_value(chunk, len(chunk))
+
+
+class Checksum(object):
+    """Hashlib-alike helper for CRC32C operations."""
+    __slots__ = ('_crc',)
+
+    def __init__(self, initial_value=b''):
+        """Construct the helper.
+
+        Args:
+            initial_value (Optional[bytes]): the initial chunk of data from
+            which the CRC32C checksum is computed.  Defaults to b''.
+        """
+        self._crc = value(initial_value)
+
+    def __int__(self):
```

Comment:
Why is this needed?

Reply:
I'm using it for test assertions.  I can use the private variable instead, if you prefer.

Reply:
89a8520